examples/groupchat/README.md

```diff
@@ -0,0 +1,14 @@
+# Multi-Agent Group Conversation in AgentScope
+
+This example demonstrates a multi-agent group conversation using AgentScope. The script `main.py` orchestrates a group chat where a user agent and several assistant agents interact dynamically in a group conversation. The chat continues until the user types "exit", signaling the end of the session.
```

Comment:
Please add more details about 
- the "@" functionality (which is an important feature)
- the background of the conversation (e.g. the roles of participants, and the topic of the conversation)